Mathlib/Topology/EMetricSpace/Defs.lean

```diff
@@ -538,8 +537,16 @@ end Compact
 
 end EMetric
 
---namespace
-/-- We now define `EMetricSpace`, extending `PseudoEMetricSpace`. -/
+/-- Extended metric spaces, with an extended distance `edist` possibly taking the value `∞`.
+
+Each `EMetricSpace` induces a canonical `UniformSpace` and hence a canonical `TopologicalSpace`.
+This is enforced in the type class definition, by extending the `UniformSpace` structure. When
+instantiating a `PseudoEMetricSpace` structure, the uniformity fields are not necessary, they
+will be filled in by default. There is a default value for the uniformity, that can be substituted
+in cases of interest, for instance when instantiating a `PseudoEMetricSpace` structure
+on a product.
```

Comment:
The comment about instantiating a PseudoEMetric space structure appears unhelpful here, since someone reading this decl isn't trying to do that. It looks like you miscopied something?

Reply:
Same as above